MatrixSDK/Crypto/CryptoMachine/MXKeysQueryScheduler.swift

```diff
@@ -94,15 +92,18 @@ public actor MXKeysQueryScheduler<Response> {
 
     private func executeQuery(users: Set<String>) async throws -> Response {
         defer {
-            if let next = nextTask {
-                log("... query completed, starting next pending query.")
-                currentQuery = .init(users: users, task: next)
+            if let nextTask = nextTask {
+                log("... query for \(users) completed, starting next pending query.")
+                currentQuery = .init(users: nextUsers, task: nextTask)
```

Comment:
maybe is a personal thing, but I don't like inferred constructor, could we use the constructor directly?